dev-app/src/screens/CollectInputsScreen.tsx

```diff
@@ -88,6 +89,13 @@ export default function CollectInputsScreen() {
                   description:
                     'Please sign if you agree to the terms and conditions',
                   submitButtonText: 'submit signature',
+                  toggles: [
+                    {
+                      title: 'fefef',
+                      description: 'essdsde',
```

Comment:
is it okay to have placeholders here or should we switch it to real text?

Reply:
+1, we should switch to real text

Reply:
I referred to the examples in the document [https://docs.stripe.com/terminal/features/collect-inputs], no title and description. These two above were for debugging and not removed yet..